test/helpers/dom/RecoveryQuestionForm.js

```diff
@@ -3,6 +3,7 @@ define(['./Form'], function (Form) {
   var ANSWER_FIELD = 'answer';
   var SHOW_ANSWER_FIELD = 'showAnswer';
   var CLASS_SELECTOR = '.recovery-question';
+  var BACK_TO_LOGIN_BTN = 'back-button';
```

Comment:
We shouldn't just add this to the RecoveryQuestionForm (it's not the RecoveryQuestion page, is it?) - these dom helpers are 1 per page